lib/validate/validate_glyphs_url.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var ValidationError = require('../error/validation_error');
+
+module.exports = function(options) {
```

Comment:
if glyphs is set to `true`, will this still run, or will that be caught ahead of time as a wrong type?